spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  before(:each) do
+    @factory = FacilityFactory.new
+    @oregon_facilities = DmvDataService.new.or_dmv_office_locations
+    @new_york_facilities = DmvDataService.new.ny_dmv_office_locations
+    @missouri_facilities = DmvDataService.new.mo_dmv_office_locations
+  end
+
+  describe '#initialize' do
+    it 'exists' do
+      expect(@factory).to be_a(FacilityFactory)
+      expect(@oregon_facilities.sample).to be_an(Hash)
+      expect(@new_york_facilities.sample).to be_an(Hash)
+      expect(@missouri_facilities.sample).to be_an(Hash)
+    end
+  end
+
+  before(:each) do
+    @oregon = @factory.create_facilities(@oregon_facilities, :OR)
+    @new_york = @factory.create_facilities(@new_york_facilities, :NY)
+    @missouri = @factory.create_facilities(@missouri_facilities, :MO)
+  end
+
+  describe '#creates facilities' do
+    it 'returns an array of facilities' do
+      expect(@oregon).to be_an(Array)
+      expect(@new_york).to be_an(Array)
+      expect(@missouri).to be_an(Array)
+    end
+  end
+  
+  describe '#oregon processing' do
+    it 'creates oregon facility objects' do
+      expect(@oregon.first).to be_a(Facility)
+      expect(@oregon.sample).to be_a(Facility)
```

Comment:
Ooh this is interesting! I like the use of `sample` here so you can make sure you aren't getting a weird false positive by only checking `first`.